### DISCUSSION TITLE
Question Related to Timing Confict

### DISCUSSION THREAD

**Discussion by vchaurasia95** [snigi-gupta/AttendingUniversityAtBuffalo#40] [General]
Hi Snigdha (@snigi-gupta),

First of all thank you for this great in-depth detailed repo.

While planning which subjects to choose this semester, I noticed there was a conflict of 10 minutes i.e: The first one finishes at 2:00 PM while the other one starts at 1:50PM. I just wanted to ask what would you suggest ? Should I opt for these both or swap in one of the courses in order to resolve the conflict. The main reason of seeking your suggestion is I was going through the classes offered and was having a hard time to find any alternatives.

Looking forward for your response.

**Comment by snigi-gupta** [comment]
Hi @vchaurasia95,

I apologize for the late response. I think so if either of them is recorded, then you can pick both, but AFAIK, the portal (HUB) will not let you chose those courses due to the overlap. Check with the department coordinator (Clara).
I can help you in choosing the subjects, if you share me your take so far and probably will be able to guide you. 

Feel free to post more questions, if you have any!

**Reply by vchaurasia95** [reply]
Hey @snigi-gupta ,

Thank you for reverting back. None of the courses are recorded, however I asked for the permission from one of the instructors (Dr. Srihari) and she was ok with me leaving 10 minutes early so I registered for the course (through Clara). Regarding the course selection I enrolled in the following courses as of now:

1. Large Scale Distributed System (Dr. Ramamurty)
2. Information Retrival (Dr. Rohini Srihari)
3. Analysis of Algorithms (Dr. He)
4. Introduction to Machine Learning (Dr. Mingchen Gao)

I was wanting to complete the Breadth Requirement this Semester hence opted for the above mentioned courses. It would be very helpful if you could provide some feedback regarding the above course selections and whether the course load is easily manageable taken together, keeping in mind that my current experience covers most of the topic covered in the Distributed System Course.

Additionally I'm also considering Computer Vision and Image Processing by Junsong Yuan if the conflict thing dosen't workout and I have to drop Algorithms this semester. Can you provide any feedback on this course also?

Looking forward for your response and thank you again for your help.
Regards,
Vishal Chaurasia

**Reply by snigi-gupta** [reply]
Hi @vchaurasia95,

This seems like a medium load keeping in mind that DS will be fairly easy for due to your familiarity with the course. I would suggest you go with this combination if you feel strongly about this. I took IR and Algo by these profs, and I liked these subjects. Go through my [IR repo here](https://github.com/snigi-gupta/Information-Retrieval-CSE-535) for more insights. I took ML by Srihari, but Mingchen Gao is the professor I would suggest too.

I just shared some info on CVIP here #44 , check that out

I hope this helps. :)
